app/assets/javascripts/Annotations/pdf_annotation_manager.js

```diff
@@ -352,6 +404,30 @@
     };
   }
 
+  /**
+   * Returns the selection point in percentage units for a touch event on
+   * a element.
+   *
+   * @param {Touch}                 touch       The touch object from the event.
+   * @param {String|DOMNode|jQuery} relativeTo  The element to calculate the offset for.
+   * @return {{x: number, y:number}}  The relative point in the element the touch occurred in.
+   */
+  function getRelativePointForTouchEvent(touch, relativeTo) {
```

Comment:
This function isn't necessary, I suggest you rename the existing mouse event function into something like `getRelativePointForEvent` and call it in both places. For the touch version, it's possible to pass `undefined` as the `mouseOffset` argument, which will result in the behaviour you're looking for.